src/utils.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Converts date values in an object to UTC, based on list of keys provided
+ * @param {*} object Input object to process
+ * @param {*} keys Keys that have date values in object
```

Comment:
```suggestion
 * @param {string[]} keys Keys that have date values in object
```
This maybe more narrower type that can be used

Reply:
Added a commit as per suggestion, and a little more. A shame that we can't restrict it to that because the file extensions are .js :")